docs/getting-started.md

```diff
@@ -2,12 +2,75 @@
 title: Getting started
 ---
 
+## Prerequisites
+
+Before you begin, make sure your development environment includes Node.js® and an npm package manager.
+
+## Step 1: Install the TypeGraphQL
+
+First, we need to install the dependency on TypeGraphQL.
+
+```node
+npm install type-graphql --save
```

Comment:
There's no need to append `--save` to the install command - it can be just `npm i type-graphql`